src/mono/mono/utils/mono-md5.c

```diff
@@ -109,11 +103,6 @@ mono_md5_init (MonoMD5Context *ctx)
 	
 	ctx->bits[0] = 0;
 	ctx->bits[1] = 0;
-	
-	if (IS_BIG_ENDIAN())	
-		ctx->doByteReverse = 1;		
-	else 
-		ctx->doByteReverse = 0;	
 }
```

Comment:
Just wondering: why did you drop BE support? 🙂 I know it's not supported officially but what about the community support?

Reply:
Ah I see, you didn't